samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/service/impl/config/ServiceConfig.java

```diff
@@ -32,23 +32,29 @@
 import io.oasp.module.rest.service.impl.RestServiceExceptionFacade;
 import io.oasp.module.rest.service.impl.json.ObjectMapperFactory;
 
+/**
+ * {@link Configuration} for (REST or SOAP) services using CXF.
+ */
 @Configuration
 @EnableWs
-@ImportResource({ "classpath:META-INF/cxf/cxf.xml" /* , "classpath:META-INF/cxf/cxf-servlet.xml" */ })
-public class ServiceConfiguration extends WsConfigurerAdapter {
+@ImportResource({ "classpath:META-INF/cxf/cxf.xml" })
+public class ServiceConfig extends WsConfigurerAdapter {
 
   /** Logger instance. */
-  private static final Logger LOG = LoggerFactory.getLogger(ServiceConfiguration.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ServiceConfig.class);
 
-  public static final String URL_PATH_SERVICES = "/services";
+  /** The services "folder" of an URL. */
+  public static final String URL_FOLDER_SERVICES = "services";
 
-  public static final String URL_FOLDER_REST = "/rest";
+  public static final String URL_PATH_SERVICES = "/" + URL_FOLDER_SERVICES;
 
-  public static final String URL_FOLDER_WEB_SERVICES = "/ws";
+  public static final String URL_FOLDER_REST = "rest";
```

Comment:
After removal of the slash in `URL_FOLDER_REST` the service is not longer available. Since CXF is poorly documented and digging through the call stack (have a look at `public Server jaxRsServer()`) did not yield any useful information, I'll revert this change in another commit.

Unfortunately, even subsystem tests do not show this issue (so, running `mvn install` works fine). Therefore, we have to investigate how the test configuration deviates from the original configration. I'll file a new issue regarding this topic.